src/main/scala/edu/ie3/simona/event/listener/ResultEventListener.scala

```diff
@@ -287,10 +295,6 @@ class ResultEventListener(
       stash()
       stay()
 
-    case Event(baseData: BaseData, UninitializedData) =>
-      unstashAll()
-      goto(Idle) using baseData
-
     case Event(Init, _) =>
```

Comment:
What's the reasoning behind this deletion?

Reply:
This has become obsolete due to the result of the sink initialization now being piped to itself